core/criteria/src/main/java/org/opennms/core/criteria/restrictions/Restrictions.java

```diff
@@ -104,6 +104,10 @@ public static AllRestriction and(final Restriction... restrictions) {
         return new AllRestriction(restrictions);
     }
 
+    public static AllRestriction multipleAnd(final Restriction... restrictions) {
+        return new AllRestriction(true, restrictions);
+    }
+
```

Comment:
If we allow `multiAnd` only on top-level, should we just remove this method?

Reply:
no, because that being used in multiAnd() method of CriteriaBuilder.java class

Reply:
Than make this thing package private or encapsulate it in another way.